src/Codeception/Module/WebDriver.php

```diff
@@ -304,13 +304,11 @@ public function _initialize()
                 $this->httpProxyPort
             );
             $this->sessions[] = $this->_backupSession();
-        } catch (WebDriverCurlException $e) {
-            throw new ConnectionException(
-                $e->getMessage() . "\n \nPlease make sure that Selenium Server or PhantomJS is running."
-            );
+            $this->webDriver->manage()->timeouts()->implicitlyWait($this->config['wait']);
+            $this->initialWindowSize();
```

Comment:
These 2 lines was already existing but I put them in the try to catch exception from Selenium.